docker/grafana/provisioning/datasources/prometheus.yml

```diff
@@ -1,11 +1,11 @@
 apiVersion: 1
 
 deleteDatasources:
-  - name: Prometheus
+  - name: Dashboard1
     orgId: 1
 
 datasources:
-  - name: Prometheus
+  - name: Dashboard1
```

Comment:
IMHO this name is a little bit misleading (this is the [canonical example](https://grafana.com/docs/grafana/next/datasources/prometheus/#provision-the-prometheus-data-source)).
If your concern is about adding more sources, what about naming it "prom server 1" or if we take into account the port that they're listening "prom server 9090" ?

Reply:
In Nautilus/RHCS 4.x this data source was named 'Dashboard', and in Pacific/RHCS 5.x this became Dashboard1. After [this PR](https://github.com/ceph/ceph/pull/41598) has been merged, Dashboard front-end will pass a `var-datasource` via URL, so this won't work. If we want to add a level of customization that allows users to change this, I'm ok with it (perhaps over-customization), but this needs to be changed now.